packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -1419,3 +1531,68 @@ class _RenderRangeSlider extends RenderBox with RelayoutWhenSystemFontsChangeMix
     return (value - _semanticActionUnit).clamp(0.0, 1.0) as double;
   }
 }
+
+
+class _ValueIndicatorRenderObjectWidget extends LeafRenderObjectWidget {
+  const _ValueIndicatorRenderObjectWidget({
+    this.valueIndicatorAnimation,
```

Comment:
I think this is just the activationAnimation on the slider state, so it probably does not need to be passed in